packages/codegen-ui-react/lib/forms/form-renderer-helper/form-state.ts

```diff
@@ -30,9 +30,15 @@ import {
   PropertyAssignment,
   PropertyName,
 } from 'typescript';
+import { lowerCaseFirst } from '../../helpers';
 
 export const getCurrentValueName = (fieldName: string) => `current${capitalizeFirstLetter(fieldName)}Value`;
 
+export const getCurrentDisplayValueName = (fieldName: string) =>
+  `current${capitalizeFirstLetter(fieldName)}DisplayValue`;
```

Comment:
The sanitization logic has now been changed to convert to camelCase, so here, it will effectively be pascal case. It's definitely best effort though. If a user has a field named `descriptionofcolor` instead of something like `description-of-color`, it will remain all lower cased.

Reply:
Best effort is fine 😃